docs/product-updates/2023-06-07-api-branch-management.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Simplify Your API Workflow with Bump.sh's API New Branch Management Feature
+tags: [New, Bump API]
+image: /files/changelog/temporary-pic.png
+---
+
+[NEED A VISUAL HERE]
+
+[Branching at Bump.sh](https://docs.bump.sh/help/branching/) makes your API version workflow easier.
+
+## Latest release
+
+And now, Branch Management is at hand with the [Bump API branching endpoints](https://developers.bump.sh/group/endpoint-branches).
+Designed to enhance your API development process, these new endpoints gives you the opportunity to boost your CI deployments. With the introduction of branch creation, listing, setting default branch, and branch deletion. Bump.sh revolutionizes how you deliver ephemeral versions and begin new branches with major versions.
+
+## Let's explore how this new feature simplifies your API workflow
+
+Creating, managing, and tracking branches is an integral part of API development. Bump.sh's Branch Management feature makes it easier than ever to handle these tasks effectively. Let's take a closer look at the key endpoints:
+
+1.[Branch creation](https://developers.bump.sh/operation/operation-post-docs-parameter-branches):
+With a single API call, you can now create a new branch effortlessly. Specify the desired branch name, and Bump.sh takes care of the rest. This streamlined process eliminates the need for manual branch creation, saving you valuable time and effort.
+2. [List of Branches](https://developers.bump.sh/operation/operation-post-docs-parameter-branches):
+Get an overview of all branches associated with your API using the list of branches endpoint. This feature provides a clear and concise view of your branch hierarchy, enabling you to manage and organize them efficiently.
+3. [Set Default Branch](https://developers.bump.sh/operation/operation-patch-docs-parameter-branches-parameter-set_default):
+Setting a default branch for your API documentation is now a breeze. Choose the branch you want to showcase as the default version, and Bump.sh ensures that users accessing your API documentation are directed to the most up-to-date and relevant information.
+4. [Delete Branch](https://developers.bump.sh/operation/operation-delete-docs-parameter-branches-parameter):
+When a branch is no longer needed or has served its purpose, you can easily delete it using the delete branch endpoint. By removing unnecessary branches, you maintain a clean and organized development environment.
```

Comment:
At first, I would have removed these points, but it does make sense to present the different operations. Maybe we could just have a simple bullet points list with a simple sentence for each point?